src/map/skill.cpp

```diff
@@ -1909,10 +1913,6 @@ int skill_additional_effect( struct block_list* src, struct block_list *bl, uint
 				skill_break_equip(src, bl, EQP_WEAPON, rate * 100, BCT_ENEMY);
 				break;
 		}
-		} break;
-
-	case LG_MOONSLASHER:
-		sc_start(src,src,SC_OVERBRANDREADY,100,skill_lv,skill_get_time2(skill_id,skill_lv));
 		break;
```

Comment:
```suggestion
		} break;
```
To fix the compile error, or at least one I saw so far!